config/routes.rb

```diff
@@ -21,7 +21,13 @@
     end
     resources :placements_preview, only: [:show]
     get "rollups_demo", to: "podcasts#rollups_demo"
-    resource :metrics, controller: :podcast_metrics
+    resource :metrics, controller: :podcast_metrics do
```

Comment:
Should probably throw an `only: :show` on here.  So you don't get the full CRUD routes.